trillian/testdata/Makefile

```diff
@@ -111,6 +111,29 @@ distclean: clean
 	rm -f $(SERVER_PUBKEYS) fake-ca.cert int-ca.cert int-ca.csr.pem
 	rm -f $(LEAF_CERTS) $(LEAF_CSRS) $(LEAF_CHAINS)
 
+
+# We also use a second fake CA as a trust root for CT tests. This is its private key.
+CA_1_PRIVKEY=fake-ca1.privkey.pem
```

Comment:
nit: consistent basenames would be nice (`fake-ca1` vs `fake-ca-1`)

Reply:
Done